### PR TITLE
Deprecate Recorded Future v1

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/content/pull/12575

## Description
Deprecate Recorded future v1, update README file with the relevant commands from v2.

## Minimum version of Cortex XSOAR
- [x] 5.0.0

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Recorded_Future/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Recorded Future (Deprecated)
+- The ***Recorded Future*** integration is deprecated. Please use ***Recorded Future v2***.
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/ReleaseNotes/1_1_0.md:4]:
```suggestion
- The ***Recorded Future*** integration is deprecated. Use ***Recorded Future v2*** from <pack name> pack.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml [hunk 2]
```diff
@@ -74,10 +74,11 @@ configuration:
   name: incidentType
   required: false
   type: 13
-description: Unique threat intel technology that automatically serves up relevant
+description: Deprecated. Use Recorded Future v2 instead. Unique threat intel technology that automatically serves up relevant
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml:77]:
```suggestion
description: Deprecated. Use Recorded Future v2 from <pack name> pack instead. Unique threat intel technology that automatically serves up relevant
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/README.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <!-- HTML_DOC -->
 <div class="cl-preview-section">
+<p>Deprecated. Use Recorded Future v2 instead.</p>
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/README.md:3]:
```suggestion
<p>Deprecated. Use Recorded Future v2 from <pack name> pack instead.</p>
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/README.md [hunk 4]
```diff
@@ -1536,6 +1546,9 @@
 </div>
 <div class="cl-preview-section">
 <h3 id="get-information-for-a-url">4. Get information for a URL</h3>
+<p>This command is deprecated.</p>
+<p>Use `recordedfuture-intelligence` command from Recorded Future v2 with the following argument `entity_type=url` and `entity=url_value`.</p>
+<p>You can find context differences in <a href="https://xsoar.pan.dev/docs/reference/integrations/recorded-future-v2#recordedfuture-intelligence">here</a>. Notice that we are no longer support community notes field.</p>
```
Comment (ShahafBenYakir) [inline Packs/Recorded_Future/Integrations/Recorded_Future/README.md:1551]: Maybe change to this?
```suggestion
<p>You can find context differences in <a href="https://xsoar.pan.dev/docs/reference/integrations/recorded-future-v2#recordedfuture-intelligence">here</a>. Notice that 'community notes' output is no longer supprted.</p>
```

**Comment by barchen1** [comment]
@amshamah419 Can you please go over the [README.md file](https://github.com/demisto/content/pull/12575/files#diff-8e19c67b167b769be8138402b2158616f975757ee0f8e6ad6901806af78bf818) and the rn?

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/Recorded_Future/ReleaseNotes/1_1_0.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Recorded Future (Deprecated)
+- The ***Recorded Future*** integration is deprecated. Use ***Recorded Future v2*** from RecordedFuture pack.
```
Comment (amshamah419) [inline Packs/Recorded_Future/ReleaseNotes/1_1_0.md:4]:
```suggestion
- The ***Recorded Future*** integration is deprecated. Use ***Recorded Future v2*** from the RecordedFuture pack.
```

File: Packs/Recorded_Future/Integrations/Recorded_Future/README.md [hunk 6]
```diff
@@ -4332,6 +4374,9 @@
 </div>
 <div class="cl-preview-section">
 <h3 id="get-a-list-of-alerts">22. Get a list of alerts</h3>
+<p>This command is deprecated.</p>
+<p>Use `recordedfuture-alerts` command from Recorded Future v2 instead.</p>
+<p>You can find context differences in <a href="https://xsoar.pan.dev/docs/reference/integrations/recorded-future-v2#recordedfuture-alerts">here</a>.</p>
```
Comment (amshamah419) [inline Packs/Recorded_Future/Integrations/Recorded_Future/README.md:4379]:
```suggestion
<p>You can find context differences <a href="https://xsoar.pan.dev/docs/reference/integrations/recorded-future-v2#recordedfuture-alerts">here</a>.</p>
```
Remove the `in` for the others as well.

**Pull request merged**